apps/admin-dashboard/app/routes/_dashboard.schools.chapters.$id.create.tsx

```diff
@@ -0,0 +1,84 @@
+import {
```

Comment:
Let's change the name of this file to '_dashboard.schools.$id.chapter.create.tsx'. See the other comment below